### ISSUE TITLE
vlc: program does not shutdown (AppArmor)

### ISSUE THREAD

**Issue by smitsohu** [netblue30/firejail#5572]
### Description

VLC does not terminate when it is closed.

### Steps to Reproduce

Start VLC, then close it. Verify that the sandbox is still alive using `firejail --tree`.

```
$ firejail --tree
3208860:smitsohu::firejail vlc
  3208861:smitsohu::/usr/bin/xdg-dbus-proxy --fd=4 --args=5
  3208863:smitsohu::firejail vlc
    3208878:smitsohu::vlc
```

### Behavior without a profile

VLC terminates when it is closed.

### Additional context

From the syslog:
```
Jan 05 17:54:25 host audit[3208878]: AVC apparmor="DENIED" operation="signal" profile="firejail-default" pid=3208878 comm="vlc" requested_mask="send" denied_mask="send" signal=term peer="firejail-default//&unconfined"
```

`firejail --ignore=apparmor vlc` behaves as expected.

`--deterministic-shutdown` is no workaround, because apparently the issue is in VLC itself.

### Environment

$ cat /etc/debian_version
11.6

### Checklist
- [x] The issues is caused by firejail (i.e. running the program by path (e.g. `/usr/bin/vlc`) "fixes" it).
- [x] I can reproduce the issue without custom modifications (e.g. globals.local).
- [x] The program has a profile. (If not, request one in `https://github.com/netblue30/firejail/issues/1139`)
- [x] The profile (and redirect profile if exists) hasn't already been fixed [upstream](https://github.com/netblue30/firejail/tree/master/etc).
- [x] I have performed a short search for similar issues (to avoid opening a duplicate).

**Comment by smitsohu** [comment]
Sorry, this has been fixed already in #5317.

Instead, there is a small bug in the Makefile, that installs the `firejail-default` apparmor profile to `/usr/local/etc/apprmor.d` by default, where the AppArmor scripts won't find it.

Closing again.

**Comment by kmk3** [comment]
(Re-closing as "not planned" since nothing was changed in firejail)

**Comment by kmk3** [comment]
Fixed by #5317.
